templates/includes/sidebar.html

```diff
@@ -0,0 +1,14 @@
+ <ul>
+    <li>
+        <a href="#">Home page</a>
```

Comment:
The href attribute currently contains a placeholder '#'. Make sure to replace it with the actual URL or route for the Home page.